Python/import.c

```diff
@@ -2964,6 +2978,32 @@ PyImport_AppendInittab(const char *name, PyObject* (*initfunc)(void))
     return PyImport_ExtendInittab(newtab);
 }
 
+
+static bool
+_PyImport_LazyImportsIsEnabled()
+{
+    PyInterpreterState *interp = _PyInterpreterState_GET();
+    if (_PyInterpreterState_GetConfig(interp)->lazy_imports |
+        interp->lazy_imports_enabled)
```

Comment:
Maybe check `interp->lazy_imports_enabled` first and use `||` here. This way it won't call `_PyInterpreterState_GetConfig()` if `lazy_imports_enabled` is set.